pkg/console/subresource/configmap/configmap_test.go

```diff
@@ -108,6 +156,7 @@ customization:
 						Host: host,
 					},
 				},
+				useDefaultCAFile: true,
```

Comment:
We have at least one that verifies this generates correctly with `useDefaultCAFile: true`?